typescript/cli/src/config/warp.ts

```diff
@@ -278,11 +279,25 @@ export async function createWarpRouteDeployConfig({
         tokenMetadata?.symbol,
         'Error deriving token metadata, please check the provided token addresses',
       );
+
+      const symbol = tokenMetadata.symbol;
+      let warpRouteId;
+      if (!context.skipConfirmation) {
+        warpRouteId = await getWarpRouteIdFromWarpDeployConfig(
+          context.registry,
+          warpRouteDeployConfig,
+          symbol,
+        );
+      }
+
       await context.registry.addWarpRouteConfig(warpRouteDeployConfig, {
```

Comment:
i was thinking that we can just rely on addWarpRouteConfig() throwing and continously loop-prompting the user until they provide the correct warpRouteId, but it's [not possible because MergedRegistry catches and logs the error](https://github.com/hyperlane-xyz/hyperlane-registry/blob/main/src/registry/MergedRegistry.ts#L165-L166)